### ISSUE TITLE
Server Side Implementaion & Reverse Tunnel

### ISSUE THREAD

**Issue by edescourtis** [drowzy/ssh_tunnel#1]
It would be nice to have both the client and server side as well as reverse tunnels. I realize that is a lot of work but it would open up many use cases.

**Comment by drowzy** [comment]
Yes I agree with you. I'll begin by adding reverse tunnel for the client and move on to implementing server capabilities after that.
Good suggestion! Thank you!
